ocs_ci/ocs/resources/pvc.py

```diff
@@ -184,6 +186,30 @@ def get_attached_pods(self):
                 attached_pods.append(pod_obj)
         return attached_pods
 
+    def create_pvc_clone(
```

Comment:
please see https://github.com/red-hat-storage/ocs-ci/pull/2713/files#r471266511.
Better to move this method to be a standalone function outside of PVC as it instantiates a PVC object

Reply:
Hi @ebenahar , I moved it to PVC class as per the suggestion by @jilju here https://github.com/red-hat-storage/ocs-ci/pull/2680#discussion_r469050118

Reply:
If it is still mandatory, please let me know :)

Reply:
I guess @ebenahar wants you to keep the function in the same file but move out of the class.

Reply:
indeed :)